lighthouse-core/audits/byte-efficiency/uses-optimized-images.js

```diff
@@ -40,7 +40,7 @@ class UsesOptimizedImages extends ByteEfficiencyAudit {
     return {
       category: 'Images',
       name: 'uses-optimized-images',
-      description: 'Unoptimized images',
+      description: 'Optimize images',
```

Comment:
i had mocked this section out with titles more in line with "Your problems" rather than the "Suggested solutions" you do here.

but since the big communication here is YOUR SAVINGS, it seems good to sit those numbers next to the action you must take.

sgtm